src/main/java/mate/academy/Main.java

```diff
@@ -2,11 +2,12 @@
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Main {
     public static void main(String[] args) {
         // Feel free to play with AsyncRequestProcessor in this main method if you want
-        ExecutorService executor = null; // Provide implementation that fits your needs
+        ExecutorService executor = Executors.newSingleThreadExecutor();
```

Comment:
Consider using a thread pool with more threads or a different type of executor. Using a single-threaded executor means that requests will be processed sequentially rather than concurrently, which may not be suitable for an asynchronous processing scenario.